test/spec_multipart.rb

```diff
@@ -871,11 +871,34 @@ def initialize(*)
     params['profile']['bio'].must_include 'hello'
   end
 
-  it "parses very long unquoted multipart file names" do
+  ['', '"'].each do |quote_char|
+    it "parses very long #{'un' if quote_char.empty?}quoted multipart file names" do
```

Comment:
In what case is `quote_char` empty?

Reply:
The first case.  `''` is empty, `'"'` is not empty.

Reply:
Hah I read it as a single quote :)